pymc/logprob/transform_value.py

```diff
@@ -97,7 +97,26 @@ def transformed_value_logprob(op, values, *rv_outs, use_jacobian=True, **kwargs)
     This is introduced by the `TransformValuesRewrite`
     """
     rv_op = rv_outs[0].owner.op
+    transforms = op.transforms
     rv_inputs = rv_outs[0].owner.inputs
+
+    if use_jacobian and len(values) == 1 and len(transforms) == 1:
+        # Check if there's a specialized transform logp implemented
+        [value] = values
+        assert isinstance(value.owner.op, TransformedValue)
+        unconstrained_value = value.owner.inputs[1]
+        [transform] = transforms
+        try:
+            return _transformed_logprob(
+                rv_op,
+                transform,
+                unconstrained_value=unconstrained_value,
```

Comment:
It would be nice if we could also pass in the transformed value. That way we can avoid computing it twice in a graph if the logp can use that value too. I guess pytensor might get rid of that duplication anyway, but I don't know how reliable that is if the transformation is doing something more complicated.

Reply:
Actually we have the constrained value as well, its `value.owner.inputs[0]`